### PR TITLE
Bug 1000550 - Make warnings less stern, where applicable

### PR DESCRIPTION
**Pull request by kmaglione**
CC @jvillalobos

### CODE REVIEW

File: validator/constants.py [hunk 1]
```diff
@@ -48,7 +48,7 @@
 
 DEFAULT_WEBAPP_MRKT_URLS = ["https://marketplace.mozilla.org",
                             "https://marketplace-dev.allizom.org"]
-BUGZILLA_BUG = "https://bugzilla.mozilla.org/show_bug.cgi?id=%d"
+BUGZILLA_BUG = "https://bugzil.la/%d"
```
Comment (magopian) [inline validator/constants.py]: Why this change?

File: validator/testcases/javascript/call_definitions.py [hunk 2]
```diff
@@ -30,10 +30,9 @@ def amp_rp_bug660359(wrapper, arguments, traverser):
                 "amp_rp_bug660359"),
         notice="Custom add-on types may not work properly in Gecko 6",
         description="This add-on appears to register custom add-on types, "
-                    "which are affected and may not work properly due to "
+                    "which are affected by, and may not work properly due to, "
                     "changes made on Gecko 6. For more information, "
-                    "please refer to "
-                    "https://bugzilla.mozilla.org/show_bug.cgi?id=595848",
+                    "please refer to %s" % BUGZILLA_BUG % 595848,
```
Comment (magopian) [inline validator/testcases/javascript/call_definitions.py]: Nice, much better to factor out the bugzilla url, thanks!

File: validator/testcases/javascript/traverser.py [hunk 3]
```diff
@@ -282,18 +282,26 @@ def _build_global(self, name, entity):
...
+                elif isinstance(dang, dict):
+                    kwargs.update(dang)
+                else:
+                    kwargs["description"] = ("Access to the `%s` property is "
+                                             "deprecated for security or "
+                                             "other reasons." % name)
+
+                self._debug("DANGEROUS")
+                self.err.warning(**kwargs)
+
```
Comment (magopian) [inline validator/testcases/javascript/traverser.py]: Much more readable that way, thanks

**Comment by magopian** [comment]
lgtm, should I wait for @jvillalobos to r+ before merging?

**Comment by jvillalobos** [comment]
The changes look good to me.

File: validator/constants.py (hunk 1 above)
Comment (kmaglione) [inline validator/constants.py]: Because the full URLs are visible in the validation messages, and the longer URLs add a lot of noise and distracting line breaks.

Comment (magopian) [inline validator/constants.py]: Makes sense, thanks for the explanation
